packages/digitalriver-integration/src/digitalriver-payment-strategy.ts

```diff
@@ -43,32 +44,29 @@ export default class DigitalRiverPaymentStrategy implements PaymentStrategy {
     private _digitalRiverCheckoutData?: DigitalRiverInitializeToken;
     private _unsubscribe?: () => void;
     private _digitalRiverInitializeOptions?: DigitalRiverPaymentInitializeOptions;
-
     constructor(
-        private _store: CheckoutStore,
-        private _paymentMethodActionCreator: PaymentMethodActionCreator,
-        private _orderActionCreator: OrderActionCreator,
-        private _paymentActionCreator: PaymentActionCreator,
-        private _storeCreditActionCreator: StoreCreditActionCreator,
+        private _paymentIntegrationService: PaymentIntegrationService,
```

Comment:
Need to get some clarification about using an underscore as the first char of a variable in the new packages.

As fas as i remember we have to wright method and variable names without them.

@animesh1987